rdr_service/cron_prod.yaml

```diff
@@ -80,3 +79,8 @@ cron:
   timezone: America/New_York
   schedule: every day 08:15
   target: offline
+- description: Ce Health Data Reconciliation Pipeline
+  url: /offline/CeHealthDataReconciliation
+  timezone: America/New_York
+  schedule: every day 3:45
+  target: offline
```

Comment:
This cron job would already be scheduled by an entry in the default file (https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/cron_default.yaml#L177). Does it need to be redefined here?

Reply:
It could be removed.